### PR TITLE
Cleaning up the repo and removing the runtime

### PR DESCRIPTION
**Pull request by SantosGuillamot**
This PR aims to clean up the movies repo and remove the code related to the runtime, as we will require the [BHE plugin](https://github.com/WordPress/block-hydration-experiments/), which should be the one requiring it. As it is right now, it is relying on the [`build-to-globals` branch](https://github.com/WordPress/block-hydration-experiments/tree/build-to-globals), as it is exposing the functions needed by the blocks in the `window`. 

Some of the things covered here:

- Require the BHE plugin.
- Remove the `runtime` and the client-side navigation files, as well as the settings. This should be handled by the other plugin.
- Remove old `npm` dependencies.
- Change the imports of the `store` and `navigate` functions to read from `window.__experimentalInteractivity`.
- Remove the `content-creator` folder, as we are generating the content in a different way.
- I also took the opportunity to add the `state` and the `selectors` to the SSR files.

### CODE REVIEW

**Comment by SantosGuillamot** [comment]
I've tried adding the BHE repo to `wp-env`, but it returns some error. Maybe it's better to wait until we create a downloadable ZIP file, similar to [this PR](https://github.com/c4rl0sbr4v0/wp-movies-demo/pull/20), so we can link directly there.

**Comment by SantosGuillamot** [comment]
> I've tried adding the BHE repo to wp-env, but it returns some error. Maybe it's better to wait until we create a downloadable ZIP file, similar to https://github.com/c4rl0sbr4v0/wp-movies-demo/pull/20, so we can link directly there.

I solved this issue in the release-action PR, which is intended to be merged in [this branch](https://github.com/c4rl0sbr4v0/wp-movies-demo/pull/20).

File: src/blocks/movie-search/view.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import { wpx } from '../../../lib/runtime/wpx.js';
-import { navigate } from '../../../lib/runtime/router.js';
+const { store, navigate } = window.__experimentalInteractivity;
```
Comment (luisherranz) [inline src/blocks/movie-search/view.js:1]:
```suggestion
// Disclaimer: Importing the `store` using a global is just a temporary solution.
const { store, navigate } = window.__experimentalInteractivity;
```

File: src/blocks/movie-tabs/view.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import { wpx } from '../../../lib/runtime/wpx.js';
+const { store } = window.__experimentalInteractivity;
```
Comment (luisherranz) [inline src/blocks/movie-tabs/view.js:1]:
```suggestion
// Disclaimer: Importing the `store` using a global is just a temporary solution.
const { store } = window.__experimentalInteractivity;
```

File: src/blocks/video-player/view.js [hunk 3]
```diff
@@ -1,11 +1,6 @@
-import { wpx } from '../../../lib/runtime/wpx.js';
+const { store } = window.__experimentalInteractivity;
```
Comment (luisherranz) [inline src/blocks/video-player/view.js:1]:
```suggestion
// Disclaimer: Importing the `store` using a global is just a temporary solution.
const { store } = window.__experimentalInteractivity;
```

**Review by luisherranz — COMMENTED** [review]
I thought we could add a disclaimer to indicate that import `store` with a global is just a temporary solution.

**Comment by michalczaplinski** [comment]
I didn't get to review it properly today, sorry. I ll get round to it on Monday 🙂

**Comment by SantosGuillamot** [comment]
> I thought we could add a disclaimer to indicate that import store with a global is just a temporary solution.

Totally makes sense 🙂 I've just committed the suggestions.

**Comment by michalczaplinski** [comment]
OK, I've pushed a few small improvements and I think we can merge this and https://github.com/c4rl0sbr4v0/wp-movies-demo/pull/20 👍👍👍

**Pull request merged**